app/views/bill-runs/match-details.njk

```diff
@@ -0,0 +1,208 @@
+        }) }}
+      </p>
+
+      {# ------------- Licence nav bars --------------- #}
+      <ul class="govuk-summary-list__actions-list govuk-!-margin-bottom-3">
+        <li class="govuk-summary-list__actions-list-item"><a class="govuk-link govuk-!-font-weight-bold govuk-link--no-visited-state govuk-link--no-underline" href="/licences/{{ licenceId }}#summary">
+            Summary
+          </a></li>
+        <li class="govuk-summary-list__actions-list-item"><a class="govuk-link govuk-!-font-weight-bold govuk-link--no-visited-state govuk-link--no-underline" href="/licences/{{ licenceId }}#returns">
+            Returns
+          </a></li>
+          <li class="govuk-summary-list__actions-list-item"><a class="govuk-link govuk-!-font-weight-bold govuk-link--no-visited-state govuk-link--no-underline" href="/licences/{{ licenceId }}#charge">
+            Charge information
+          </a></li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="govuk-caption-m"> Financial year {{ financialYear }}</div>
+  <h2 class="govuk-heading-m">Charge period {{ chargePeriod }}</h2>
+
+  {# ---------- Billable returns and volume ----------- #}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <div class="govuk-!-margin-right-6 float-left">
+        <h3>
+          <div class="govuk-body govuk-font-size-64 govuk-!-font-weight-bold">
+            {{chargeElement.billableVolume}}ML
+          </div>
+          <div class="govuk-body govuk-!-font-weight-bold govuk-body govuk-!-font-size-19">
+            Billable returns
+          </div>
+        </h3>
+      </div>
+      <div class="govuk-!-margin-right-6 float-left">
+        <h3>
+          <div class="govuk-body govuk-font-size-64 govuk-!-font-weight-bold">
+            /
+          </div>
+        </h3>
+      </div>
+      <div class="govuk-!-margin-right-6 float-left">
+        <h3>
+          <div class="govuk-body govuk-font-size-64 govuk-!-font-weight-bold">
+            {{chargeElement.authorisedVolume}}ML
+          </div>
+          <div class="govuk-body govuk-!-font-weight-bold govuk-body govuk-!-font-size-19">
+            Authorised volume
+          </div>
+        </h3>
+      </div>
+      <div class="govuk-!-margin-right-6 float-left">
+        <h3>
+          {% if chargeElement.issues.length > 0 %}
+            <div class="govuk-inset-text">
+              <h4 class="govuk-heading-s govuk-!-font-weight-bold">
+                Issue
+              </h4>
+              {% for issue in chargeElement.issues %}
+                <div> {{ issue }} </div>
+              {% endfor %}
+            </div>
+          {% endif %}
+        </h3>
+      </div>
+    </div>
+  </div>
+
+  {# ---------- Billable returns button ----------- #}
+  <div class="govuk-grid-row govuk-!-margin-bottom-6">
+    <div class="govuk-grid-column-full">
+      <button class="govuk-button govuk-button--secondary">
+        Edit the billable returns
+      </button>
+    </div>
+  </div>
```

Comment:
Just a question. Is it not possible to use the [GOV.UK button component](https://design-system.service.gov.uk/components/button/) for this?